__tests__/app.test.js

```diff
@@ -97,7 +97,33 @@ describe('/api/articles', () => {
       });
   });
 });
-
+describe('/api/articles?topic', () => {
+  test('GET 200: responds only with articles of a given topic', () => {
+    return request(app)
+      .get('/api/articles?topic=mitch')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.length).toBe(12);
+        expect(body.articles[0].topic).toBe('mitch');
```

Comment:
I don't think that checking the first article in the array is a strong enough assertion. It would be more robust to iterate through the articles and check that all have the topic of 'mitch'